INSTALLATION.rst

```diff
@@ -47,6 +47,39 @@ must use the compilation options (see below) to specify where they are.
           This is discussed below in the context of MacOSX, where it is often the
           easiest way to get the dependencies, but it is equally applicable to linux.
 
+Ubuntu
+^^^^^^
+It is best to use a system-wide installation of ``gcc, gsl,`` and ``fftw3`` rather than installing
+them in a conda environment. ``gcc`` is by default available in Ubuntu.
+To check if ``gcc`` is installed, run ``gcc --version`` in your terminal.
+Install ``fftw3`` and ``gsl`` on your system with  ``sudo apt-get install libfftw3-dev libgsl-dev``.
+
+Once these are successfully installed, check where the ``fftw3`` library is actually located
+using ``locate libfftw3.so``. For example, it may be located in ``/usr/lib/x86_64-linux-gnu/``.
```

Comment:
I don't *think* you need to say this, because I think `apt` will always place the packages on the search path.